app/controllers/steps_controller.rb

```diff
@@ -0,0 +1,27 @@
+class StepsController < ApplicationController
+  include Wicked::Wizard::Translated
+
+  before_action :authenticate_user!
+
+  steps(*User.steps)
```

Comment:
This is a Ruby splat, so `User.steps #=> [:stage_1, stage_2, ..., stage_5]` and it tells Wicked that those are our five steps, while making it easy to add more steps on the User by adding them in the `aasm do [...] end` block.